docs/static/js/algolia.js

```diff
@@ -0,0 +1,91 @@
+// Algolia Docsearch implement
+// https://docsearch.algolia.com/docs/api
+
+const translations = {
+  placeholder: '搜索文档',
+  button: {
+    buttonText: '搜索',
+    buttonAriaLabel: '搜索',
+  },
+  modal: {
```

Comment:
Where's the translation part for `noResultsScreen` according to https://docsearch.algolia.com/docs/api/#translations